kolibri/utils/options.py

```diff
@@ -45,6 +45,29 @@ def calculate_thread_pool():
 
 
 option_spec = {
+    "Cache": {
+        "CACHE_BACKEND": {
+            "type": "option",
+            "options": ("memory", "redis"),
+            "default": "memory",
+            "envvars": ("KOLIBRI_CACHE_BACKEND",),
+        },
+        "CACHE_TIMEOUT": {
+            "type": "integer",
+            "default": 300,
+            "envvars": ("KOLIBRI_CACHE_TIMEOUT",),
+        },
+        "CACHE_PASSWORD": {
```

Comment:
Should we add an error if these aren't defined, and we're using redis?

Reply:
Maybe not for password, as it's not required, but either we do that for location, or as @jamalex suggested, we should have a default location for `localhost:6379` which is the redis default.